### PR TITLE
Updated index.html

### PR DESCRIPTION
**Pull request by frankiecoots**
### Description

Updated the <svg> element to use <use href="#link"/> instead of the deprecated attribute <use xlink:href="#link"/>

### Motivation & Context

xlink:href attribute is obsolete and deprecated

### Type of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would change existing functionality)

### Checklist

- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)
- [x] My code follows the code style of the project _(using `npm run lint`)_
- [ ] My change introduces changes to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

#### Live previews

- <https://deploy-preview-39222--twbs-bootstrap.netlify.app/>

### CODE REVIEW

**Review by julien-deramond — COMMENTED** [review]
Based on https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href, `xlink:href` seems indeed to be deprecated.

> Deprecated: This feature is no longer recommended. Though some browsers might still support it, it may have already been removed from the relevant web standards, may be in the process of being dropped, or may only be kept for compatibility purposes. Avoid using it, and update existing code if possible; see the [compatibility table](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href#browser_compatibility) at the bottom of this page to guide your decision. Be aware that this feature may cease to work at any time.

> Note: SVG 2 removed the need for the xlink namespace, so instead of xlink:href you should use [href](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/href). If you need to support earlier browser versions, the deprecated xlink:href attribute can be used as a fallback in addition to the href attribute, e.g. <use href="some-id" xlink:href="some-id" x="5" y="5" />.

[Browser compatibility page of `href`](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/href#browser_compatibility) is compatible with [our `.browserslistrc`](https://browserslist.dev/?q=Pj0gMC41JSBvciBsYXN0IDIgbWFqb3IgdmVyc2lvbnMgb3Igbm90IGRlYWQgb3IgQ2hyb21lID49IDYwIG9yIEZpcmVmb3ggPj0gNjAgb3IgRmlyZWZveCBFU1Igb3IgaU9TID49IDEyIG9yIFNhZmFyaSA%2BPSAxMiBvciBub3QgRXhwbG9yZXIgPD0gMTE%3D) except for an edge case described right after.

We support Safari for iOS from 12.0-12.1 and `href` is available from 12.2. It represents 0.017% of the overall browser coverage so I think it's rather safe to ignore this compatibility given the fact that we plan in v6 to update our browsers list compatibility.

I confirm this compatibility issue based on Browserstack tests on real devices where it's OK in Safari for iOS 13 but not in 12 where the icons are not displayed. But this is not the only issue we have in our documentation in Safari for iOS 12 where the header is not rendered for example.

In conclusion, seems to be fair to go in this direction without having to wait for our big browsers list update in v6. @twbs/css-review thoughts? Would you mind to double-check?

Note: if we're OK, this PR will need to be updated to apply this change everywhere and not only in the Jumbotrons example

**Comment by XhmikosR** [comment]
We don't change browser support in non-major versions.
